src/dmd/cppmangle.d

```diff
@@ -1081,26 +1082,24 @@ private final class CppMangleVisitor : Visitor
     }
 
     /******
-     * Write out 1 or 2 character basic type mangling.
+     * Write out basic type mangling.
      * Handle const and substitutions.
      * Params:
      *  t = type to mangle
-     *  p = if not 0, then character prefix
-     *  c = mangling character
+     *  str = mangling string
+     *  builtin = true if handling C++ builtin type
```

Comment:
I'd mention explicitly here that the `builtin` bool specifies substitution or not. To me, it is not clear whether `__float128` or `float _Complex` is a C++ builtin type or not.

Reply:
Added it in the comment above the condition.